META.list

```diff
@@ -755,23 +755,23 @@ https://raw.githubusercontent.com/supernovus/perl6-fastcgi/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-client/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-easy/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-status/master/META6.json
-https://raw.githubusercontent.com/supernovus/perl6-method-modifiers/master/META.info
+https://raw.githubusercontent.com/supernovus/perl6-method-modifiers/master/META6.json
 https://raw.githubusercontent.com/raku-community-modules/raku-mime-types/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-text-table-list/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-timezone/master/META6.json
-https://raw.githubusercontent.com/supernovus/perl6-web-app-ballet/master/META.info
+https://raw.githubusercontent.com/supernovus/perl6-web-app-ballet/master/META6.json
```

Comment:
This seems to be at https://github.com/raku-community-modules/Web-App-Ballet/blob/master/META6.json now

Reply:
Yep, all supernovus modules were moved to the community. As long as you're changing it, use the new URL. If there's some error even then, we can accept the PR (or do the change ourselves) quickly.